build/toolchains/BUILD.bazel

```diff
@@ -171,9 +191,45 @@ config_setting(
     },
 )
 
+bool_flag(
+    name = "nogo_flag",
+    build_setting_default = False,
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
```

Comment:
So, as far as I can tell, the net effect of this is that --config nolintonbuild always takes precedence over --config lintonbuild.

That means that, if I'm reading this correctly, dev doctor will NOT lint the builds of people who have specified --config nolintonbuild in their user RC.

Is that intended?

Reply:
You're right, that is not intended. :) This is fixed.

What we don't have a way to do right now is to FORCE building without `nogo` when your `.bazelrc.user` is configured to use `--config lintonbuild`. Right now I don't think that's necessary for any reason but it is something to think about.

Reply:
Makes sense to me! :)